pkg/plugins/scms/github/scm.go

```diff
@@ -65,7 +65,13 @@ func (g *Github) Commit(message string) error {
 		return err
 	}
 
-	err = git.Commit(g.Spec.User, g.Spec.Email, commitMessage, g.GetDirectory())
+	gpgSigningKey := g.Spec.GPG.SigningKey
+	gpgPassphrase := g.Spec.GPG.Passphrase
+	if !g.Spec.GPG.Enabled {
```

Comment:
I am not sure to understand the added value of the variable `g.Spec.GPG.Enabled`
I am wondering if we could just remove

Reply:
No objections removing the `g.Spec.GPG.Enabled`. I was trying to implement the idea from #274 with the key being present in config, but not enabled. Now that I think about it more, there is probably no little value in having it explicitly set to `false` while the key is still defined.

Reply:
I will work on changes to remove it.

Reply:
> No objections removing the `g.Spec.GPG.Enabled`. I was trying to implement the idea from #274 with the key being present in config, but not enabled. Now that I think about it more, there is probably no little value in having it explicitly set to `false` while the key is still defined.


Thanks for the clarification, that was my understanding as well. As @dduportal point out, the `enabled` wasn't necessarily useful and while reviewing your code, I couldn't find a motivation to keep it around.